<ui version="4.0" >
 <class>VariableDetailDialog</class>
 <widget class="QDialog" name="VariableDetailDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Variable Details</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <item row="3" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="TextLabel1" >
     <property name="text" >
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="eType" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="eName" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="TextLabel2" >
     <property name="text" >
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="TextLabel3" >
     <property name="text" >
      <string>Value:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QTextEdit" name="eValue" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
     <property name="acceptRichText" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>eName</tabstop>
  <tabstop>eType</tabstop>
  <tabstop>eValue</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VariableDetailDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>19</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>22</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VariableDetailDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>106</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel" >
     <x>106</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
